common/decrees/00_decree.txt

```diff
@@ -666,7 +667,8 @@ decree_greener_grass_campaign = {
 	}
 	modifier = {
 		# state_migration_pull_mult = 0.50 # Was 0.75 - Added in on_actions
-		state_migration_from_arable_land_mult = 0.00333
+		state_migration_from_arable_land_mult = 0.002
+		state_migration_pull_mult = 0.15
```

Comment:
ehm... what is the idea here, is the comment "added in on_actions" still correct?

Reply:
Yes. We reduced the amount scaled from arable land and added a smaller base bonus. This moderates the decree to make it still useful to help fill states with lower arable land, while its primary goal remains encouraging pops into places with lots of free land.

Reply:
Comment should be added to communicate that.